.eslintrc

```diff
@@ -55,6 +56,8 @@
     // Force IIFEs to be wrapped in parentheses.
     "wrap-iife": [2, "inside"],
 
-    "yoda": [2, "never", {"exceptRange": true}]
+    "yoda": [2, "never", {"exceptRange": true}],
+
+    "no-regex-spaces": 0
```

Comment:
Instead of changing this globally, we should just disable it for the parseUri code. See [here](http://eslint.org/docs/user-guide/configuring.html#configuring-rules).